db/migrations/20180612233338-create-installation.js

```diff
@@ -8,6 +8,9 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.INTEGER
       },
+      githubHost: {
+        type: Sequelize.STRING
```

Comment:
we can add another migration for adding a column, as above migration dated 2018 & we might get conflicts if we want to merge master in future